src/overlays/actors/ovl_En_Owl/z_en_owl.c

```diff
@@ -1,3 +1,9 @@
+/*
+ * File: z_en_owl.c
+ * Overlay: ovl_En_Owl
+ * Description: Owl
```

Comment:
```suggestion
 * Description: Kaepora Gaebora
```
Maybe, depends on if we want to count the Encyclopedia and the like as official.

Reply:
```suggestion
 * Description: Kaepora Gaebora (Owl)
```
Perhaps be as explicit as possible?